manifests/katello.pp

```diff
@@ -14,10 +15,16 @@
 
   $katello_www_pub_dir            = '/var/www/html/pub'
   $rhsm_ca_dir                    = '/etc/rhsm/ca'
+  $katello_rhsm_setup_script      = 'katello-rhsm-consumer'
+  $katello_rhsm_setup_script_location = "/usr/bin/${katello_rhsm_setup_script}"
+
   $candlepin_consumer_name        = "katello-ca-consumer-${::fqdn}"
   $candlepin_consumer_summary     = "Subscription-manager consumer certificate for Katello instance ${::fqdn}"
   $candlepin_consumer_description = 'Consumer certificate and post installation script that configures rhsm.'
 
+  $katello_default_ca_file = "${certs::pki_dir}/certs/${certs::katello::default_ca_name}.crt"
+  $katello_server_ca_file = "${certs::pki_dir}/certs/${certs::katello::server_ca_name}.crt"
```

Comment:
This is essentially redefining https://github.com/Katello/puppet-certs/blob/master/manifests/init.pp#L130 which you could just use to DRY this up.